source/helpers/render-conventional-commit-types.ts

```diff
@@ -0,0 +1,31 @@
+import zipTextNodes from 'zip-text-nodes';
+
+// Using https://www.conventionalcommits.org/ as a reference.
+
+export function getConventionalCommitAndScopeMatch(commitTitle: string): RegExpExecArray | undefined {
+	const match = /^(?<type>\w+)(?:\((?<scope>.+)\))?: .+$/.exec(commitTitle);
```

Comment:
```suggestion
	return /^(?<type>\w+)(?:\((?<scope>.+)\))?: .+$/.exec(commitTitle);
```

If it complains about `null` anywhere you can append `?? undefined` to this line.

Reply:
> Also, I mentioned this before, do not include .+$ in the regex because it’s completely unnecessary.

It was included to match only when a conventional commit is present.

`a:` is not a conventional commit. Let me know if you still want the change.